### DISCUSSION TITLE
가을타는 4주차 회고록

### DISCUSSION THREAD

**Discussion by KwonNaeHyeon** [BDD-CLUB/challenge-every-week-posting#186] [5. 회고]
[가을타는 4주차 회고룍](https://lateral-value-9a9.notion.site/15da0d7ba22f4c629e889514c4120e9b)
안녕하세요. 수요일 일요일 이틀 간 비가 쏟아지더니, 여름이 씻겨나가고 선선한 기후가 찾아왔습니다.
비록 아직 아침과 밤엔 춥고, 점심 땐 덥지만, 점심 때에도 선선해지는 것이 머지않은 것 같습니다.
이런 날씨일 수록 더더욱 감기 조심하시길 바랍니다. 감사합니다.

**Comment by pnudpwls1103** [comment]
소프트웨어공학 과제의 곤란함과 수면시간에 대한 고민에 대해 엄청 공감이 되네요.. 그 시기에 학교 할 일과 개인 할 일이 겹쳐서 수면 패턴이 흐트러지고 의지대로 잘 안 되는 시기였던거 같아요. 저도 수면 관리는 그때 잘 안 됐어서 도움이 되는 말을 못하겠네요 흑흑..

소프트웨어공학 과제는 설계하는 느낌인데 피드백이 없어서 이게 맞나..? 하는 생각이 자주 들었던거 같습니다. 지금 돌아보면 소웨공 과제가 영 감이 안 오면 하고 있는 프로젝트에 소웨공 내용을 적용하는게 훨씬 머리가 잘 돌아갈지도 몰라요..! 프로젝트의 요구사항에 대해 얼마나 잘 알고 있는지에 따라 설계 머리가 잘 돌아가는거 같은데 코레일 어플은 관심이 없어서(정확히는 그 업계에 종사하고 있지 않아서 그럴지도..) 그런지 설계를 대충하게 되더라구요.. 소웨공 과제에 대해 크게 스트레스 받기보다 일단 해보고 다른거에 적용해봐야지~ 하는 마음이 과제하는데 스트레스 덜 받지 않을까 싶네요..! 적다보니 말이 많아졌는데 화이팅!!!
